ext/php7/coms.c

```diff
@@ -1,5 +1,6 @@
 #include "coms.h"
 
+#include <SAPI.h>
```

Comment:
Is this needed for something or did it sneak in here? Not seeing any new usage in the diff.

Reply:
This is required for `sapi_module.name` which is used in the `Datadog-Meta-Lang-Interpreter` header.

Reply:
I wonder where fields like these are documented. I thought that field was things like ZendEngine vs HHVM.

Reply:
Here's where it's [documented in code](https://github.com/DataDog/datadog-agent/blob/05a509ebe00488db75969ca980f23ac21a685fb2/pkg/trace/api/api.go#L334-L336). I believe the SAPI was chosen as a best fit for this header since we don't support anything but the Zend Engine. But you're right, `Interpreter` isn't what the SAPI is. :)